### DISCUSSION TITLE
Поля ФРМО Дата удаления, Причина удаления – возможно следует реформировать

### DISCUSSION THREAD

**Discussion by eugenekogan-spb** [fhir-ru/core#43] [Административная РГ]
Сейчас в ФРМО есть поля:
Дата удаления (Дата)
Причина удаления (В связи с реорганизацией, Дубль МО, По причине некорректного занесения МО, В связи с прекращением осуществления медицинской деятельности)

В случае, когда МО прекратила деятельность, эти поля содержат информацию о МО.
В случае действующей МО эти поля содержат информацию не о МО, а служебные данные о ведении справочника (например, чистка дублей).
Кажется, следует разделить эти данные.

**Comment by niquola** [comment]
Для этого в FHIR вроде подойдет [Provenance](https://www.hl7.org/fhir/provenance.html)

**Comment by eugenekogan-spb** [comment]
Я тоже думаю что подойдет.
Тогда было бы разумно, на мой взгляд, изменить структуру ФРМО, разделив в нем информацию о медицинской организации и информацию о действиях с реестровой записью о медицинской организации.

**Reply by Vrodvi** [reply]
Правильно ли я понимаю, что объединение\разделение организаций будет означать удаление\отключение записей об организациях и заведение записей о новых организациях?

Если, к примеру, текущее юридическое лицо с тремя филиалами:
```
- ФГБУ «НМИЦ радиологии» Минздрава России
    -- МНИОИ им. П.А.Герцена - филиал ФГБУ «НМИЦ радиологии» Минздрава России
    -- НИИ урологии и интервенционной радиологии им. Н.А. Лопаткина - филиал ФГБУ «НМИЦ радиологии» Минздрава России
    -- МРНЦ им. А.Ф. Цыба - филиал ФГБУ «НМИЦ радиологии» Минздрава России
```
разделится на три отдельных юр.лица:
```
- МНИОИ им. П.А.Герцена - филиал ФГБУ «НМИЦ радиологии» Минздрава России
- НИИ урологии и интервенционной радиологии им. Н.А. Лопаткина - филиал ФГБУ «НМИЦ радиологии» Минздрава России
- МРНЦ им. А.Ф. Цыба - филиал ФГБУ «НМИЦ радиологии» Минздрава России
```
, то будет ли это означать удаление\отключение четырёх исходных организаций (головная организация и три филиала) и заведение трёх новых организаций?
OIDы по ФРМО у новых организациях сохранятся или будут присвоены новые?

Вопрос связан с поднятым вопросом о периоде действия связи между организациями.
Если нужно будет сослаться на организацию "МРНЦ им. А.Ф. Цыба - филиал ФГБУ «НМИЦ радиологии» Минздрава России" до и после разделения, то будет ли это одна и та же организация (OID тот же) или это будут две организации (два разных OIDа)?
